api/v1alpha1/edgedevice_types.go

```diff
@@ -55,6 +55,19 @@ type Retention struct {
 type Storage struct {
 	S3 *S3Storage `json:"s3,omitempty"`
 }
+
+type OsInformation struct {
+
+	//Automatically upgrade the OS image
+	AutomaticallyUpgrade bool `json:"automaticallyUpgrade,omitempty"`
```

Comment:
Do we wish to have a broader policy than 'manual/auto' ?

Reply:
Yes we do :)

Reply:
So perhaps this can be represented differently:
```
upgradePolicy: (any of automaic/scheduled)
  automaic: true
  scheduled: cron expression
```
Or in alternative way that will enable it.

Reply:
I will do it as part of the task: https://github.com/jakub-dzon/k4e-operator/pull/69#pullrequestreview-867575037